src/ol/source/Zoomify.js

```diff
@@ -253,6 +257,19 @@ class Zoomify extends TileImage {
       transition: options.transition
     });
 
+    /**
+     * @protected
```

Comment:
The renderer reads this value with every render call, which means it can be changed on the source and the change will be reflected in the rendered result. So instead of marking it `@protected` and using a getter, you can access the member directly, and get rid of the getter.

Reply:
You're right. Having `getZDirection` in the API doc would also be a little confusing.